twirp.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.required_ruby_version = '>= 1.9'
-  spec.add_runtime_dependency 'google-protobuf', '~> 3.0', '>= 3.7.0'
+  spec.add_runtime_dependency 'google-protobuf', '>= 3.7.0', '< 5'
```

This should be `>= 3.18` (~2021 era) which is the first version that supports the descriptor syntax. Ideally we would use `>= 3.25` as [grpc does](https://github.com/grpc/grpc/pull/36982).

```suggestion
  spec.add_runtime_dependency 'google-protobuf', '>= 3.25', '< 5.a'
```